test/gtest/common/test.cc

```diff
@@ -41,6 +41,8 @@ test_base::~test_base() {
                        m_state == NEW ||    /* can be skipped from a class constructor */
                        m_state == ABORTED,
                        "state=%d", m_state);
+
+    madvise(NULL, SIZE_MAX - ucs_get_page_size(), MADV_DOFORK);
```

Comment:
1. maybe cher return value (expect ENOMEM or 0)
2. shouldn't it be ucs_align_down_pow2(SIZE_MAX, ucs_get_page_size()), i think the above expression will give non-pow2 result